scripts/build_and_release_image.sh

```diff
@@ -45,20 +49,31 @@ docker build --build-arg PUBLIC_URL=/ -t frontend-awslambda .
 popd
 docker build -f Dockerfile.awslambda -t ${ECR_REPO} .
 
-# These upload the container to the public repo
 ECR_ENDPOINT="public.ecr.aws/pcm"
+
 ECR_IMAGE=${ECR_ENDPOINT}/${ECR_REPO}:${TAG}
 aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin "${ECR_ENDPOINT}"
-docker tag ${ECR_REPO} ${ECR_IMAGE}
-docker push ${ECR_IMAGE}
-echo "Uploaded: " ${ECR_IMAGE}
 
-if [ "$RELEASE_SET" == "true" ]; then
-    VERSION_TAG=`get_current_pcm_version`
-    ECR_IMAGE_VERSION_TAGGED=${ECR_ENDPOINT}/${ECR_REPO}:${VERSION_TAG}
-
-    docker tag ${ECR_REPO} ${ECR_IMAGE_VERSION_TAGGED}
-    docker push ${ECR_IMAGE_VERSION_TAGGED}
+if [ -z $TAG ]; then
+  TAG=`get_default_pcui_version`
+  echo "Using default versioning strategy, tag: $TAG" 1>&2
+elif ! [[ $TAG =~ [0-9]{4}\\.[0-9]{2} ]]; then
```

Comment:
suggestion(non-blocking): a slightly better regex to make sure the months is valid could be `\\.[0-1][0-9]`

Reply:
Nice one, I'll switch to `(0[1-9]|1[0-2])` since the second digit needs to be ≥ 1, this will validate more strictly month values